stboot.go

```diff
@@ -118,41 +155,33 @@ func main() {
 		host.Recover()
 	}
 
-	// STBOOT and STDATA partitions
-	if !*labSetup {
-		if err = host.MountBootPartition(); err != nil {
-			stlog.Error("mount STBOOT partition: %v", err)
-			host.Recover()
-		}
-		// TODO: validate expected file paths
-		if err = host.MountDataPartition(); err != nil {
-			stlog.Error("mount STDATA partition: %v", err)
-			host.Recover()
-		}
-		// TODO: validate expected file paths
-	}
-
 	// load options
 	var securityLoader, hostCfgLoader opts.Loader
 
 	securityLoader = opts.NewSecurityFile(securityConfigFile)
 
-	if *efivarHostcfg != "" {
+	switch hostCfg.location {
+	case hostCfgEfivar:
 		_, err := mount.Mount("efivarfs", "/sys/firmware/efi/efivars", "efivarfs", "", 0)
 		if err != nil {
 			stlog.Error("mounting efivarfs: %v", err)
 			host.Recover()
 		}
 		stlog.Info("mounted efivarfs at /sys/firmware/efi/efivars")
-		_, r, err := efivarfs.SimpleReadVariable(*efivarHostcfg)
+		_, r, err := efivarfs.SimpleReadVariable(hostCfg.name)
```

Comment:
What if `hostCfg.name == ""`? 
Do we have a default efivar name?

Reply:
I added an condition during parsing. name cannot be empty anymore